index.js

```diff
@@ -315,6 +318,12 @@ class K8sExecutor extends Executor {
                         `${JSON.stringify(resp.body.status, null, 2)}`);
                 }
 
+                const waitingReason = hoek.reach(resp.body, CONTAINER_WAITING_REASON_PATH);
+
+                if (waitingReason === 'ErrImagePull') {
+                    throw new Error('Build failed to start. Please check if your image is valid.');
```

Comment:
This is same message with k8s-vm
https://github.com/screwdriver-cd/hyperctl-image/blob/f44bbccb35f31b3b30469ef6f251dc74a409ff77/scripts/hyper-runner.sh#L133